config/config_center_config.go

```diff
@@ -50,3 +52,11 @@ func (c *ConfigCenterConfig) UnmarshalYAML(unmarshal func(interface{}) error) er
 	}
 	return nil
 }
+
+func (c *ConfigCenterConfig) GetUrlMap() url.Values {
+	urlMap := url.Values{}
+	urlMap.Set(constant.CONFIG_NAMESPACE_KEY, c.ConfigFile)
+	urlMap.Set(constant.CONFIG_GROUP_KEY, c.Group)
```

Comment:
Why set configFile as namespace ? How about create a new field namespace in configCenterConfig struct?

Reply:
fixed , properties item has changed to the same as dubbo